components/search/SearchSuggestion.vue

```diff
@@ -311,7 +288,6 @@ import { NeoIcon, NeoSkeleton, NeoTabItem, NeoTabs } from '@kodadot1/brick'
 import { useTopCollections } from '../landing/topCollections/utils/useTopCollections'
 import { fetchCollectionSuggestion } from './utils/collectionSearch'
 import type { DefaultCollectionSuggestion, SearchQuery } from './types'
-import { denyList } from '@/utils/constants'
```

Comment:
deny list is used on other platforms too

Reply:
yup, updated